discovery-provider/graphql/src/index.ts

```diff
@@ -0,0 +1,42 @@
+import { ApolloServer } from '@apollo/server'
+import { startStandaloneServer } from '@apollo/server/standalone'
+import { typeDefs, resolvers } from './schema'
+import knex from 'knex'
+import { DiscoveryDB } from './datasources/db'
+
+const dbConfig = {
+  client: 'pg',
+  connection: {
+    host: 'db',
+    port: 5432,
+    user: 'postgres',
+    password: 'postgres',
+    database: 'audius_discovery'
+  }
+}
```

Comment:
it may be simpler to use a connection string like our other knex/ts apps, that way if any of the configuration changes it's only in the url, plus you can gather it right from the discovery .env file

example of how the notifications plugin does it:
https://github.com/AudiusProject/audius-protocol/blob/main/discovery-provider/plugins/notifications/src/conn.ts